google/ggrc-core
# QUICK-FIX Fix automated deployments

# Issue description

ISSUE_TRACKER settings were not included into the automated deployment
script.

# Steps to test the changes

Deploy the app same as the CI server does:
1. Download the settings file for the dev instance. Let is be in `~/Downloads/dev-settings`
2. Replace `/tmpfs/src/gfile/settings` in `kokoro/ggrc_release.sh` line 6 with your `~/Downloads/dev-settings`.
3. (optional) Disable the actual deployment code from `./bin/do_deploy` (remove, comment out, early exit) so you don't mess up the server.
4. Run:
```bash
export CURRENT_SCRIPTPATH="$(pwd)/kokoro"
./kokoro/ggrc_release.sh
```
Expected result: you get to the deployment step (`Deploying src/app.yaml into version 1`).
Actual result: you fail at `Generating app.yaml` step.

# Solution description

Add the omitted variables to `ggrc_release.sh`.

# Sanity checklist




# PR Review checklist
